ci-operator/jobs/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.9-presubmits.yaml

```diff
@@ -1,7 +1,7 @@
 presubmits:
```

Comment:
commit message missing values starting + ending with values: `doc/` and `.md`

Reply:
Thanks completely missed that - fixed.